JP Localization

ChanoSan opened the pull request:
Added in localized "LokinetOverview.md" in "loki-docs/JP/Lokinet/"

ChanoSan committed:
Create LokinetOverview.md

Add JP localization of /docs/Lokinet/LokinetOverview.md

ChanoSan committed:
Merge pull request #1 from ChanoSan/JP_localization

Create LokinetOverview.md

ChanoSan committed:
Typo fix

removed space between "title:" and "Loki"

Sonofotis commented:
Thanks @ChanoSan , I'm going to leave this PR open until we can work out a way to serve the Japanese docs site.

ChanoSan commented:
Understood. Should I add newly updated translations as PRs too, or just wait on it till you sort out how to serve them?

ChanoSan committed:
Create FAQ.md

JP translation of FAQ

ChanoSan committed:
Update LokinetOverview.md

updated to match commit 228ac9b in original

ChanoSan committed:
Create lokinet-windows-guide.md

Visual assets still reference /.../docs/assets to avoid breaking pages for now, but assets directory may need to be modified when JP translations are fully deployed

ChanoSan committed:
Update lokinet-windows-guide.md

corrected brackets and punctuation

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update lokinet-windows-guide.md

Finally figured out how internal links work :p

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Create lokinet-mac-guide.md

ChanoSan committed:
Update lokinet-mac-guide.md

ChanoSan committed:
Create lokinet-linux-guide.md

ChanoSan committed:
Merge pull request #2 from loki-project/master

Updating Fork

ChanoSan committed:
Loki Messenger to Session

Update references to Session, add placeholder for future translations of Session FAQ

ChanoSan committed:
Update FAQ.md

ChanoSan committed:
Update FAQ.md

fixed formatting

ChanoSan committed:
Create HostingSNApps.md

JP translation of SNApp Hosting Guide

ChanoSan committed:
Update FAQ.md

Add Session FAQ translations

ChanoSan committed:
Merge pull request #3 from loki-project/master

Adding changes from upstream

ChanoSan committed:
Edits to match upstream changes

ChanoSan committed:
Edit: rename Loki Messenger to Session

ChanoSan committed:
Merge pull request #4 from loki-project/master

merge changes from upstream

ChanoSan committed:
Create opengroup_setup.md

asha-ndf committed:
Create AccessingSNApps.md

ChanoSan committed:
Merge pull request #5 from asha-ndf/master

Create AccessingSNApps.md

ChanoSan committed:
Merge pull request #6 from loki-project/master

merge upstream changes

ChanoSan committed:
Merge pull request #7 from loki-project/master

upstream changes

ChanoSan committed:
Create JoinOpenGroupMobile.md

ChanoSan committed:
Create JoinOpenGroupDesktop.md

ChanoSan committed:
Merge pull request #8 from loki-project/master

Merge upstream changes

ChanoSan committed:
Update HostingSNApps.md

Fix incorrect commands in line with upstream changes

ChanoSan committed:
Create Session.md

asha-ndf committed:
Update LokinetRouter.md

Some VPS hasn't root user by default. This guide is for non-developers. adduser command needs by su or root. So, I think it is better with sudo.

ChanoSan committed:
Merge pull request #9 from loki-project/master

merge upstream changes

ChanoSan committed:
Update Session.md

fix bad href to assets

ChanoSan committed:
Update FAQ.md

match upstream changes

ChanoSan committed:
Update FAQ.md

ChanoSan committed:
Create LokinetGuiGuide.md

ChanoSan committed:
Update LokinetGuiGuide.md

fix assets link

asha-ndf committed:
Add files via upload

ChanoSan committed:
Merge pull request #10 from asha-ndf/master

LokinetRouter.md

ChanoSan committed:
Merge pull request #11 from loki-project/master

Add hyperlink to evoultion host

ChanoSan committed:
Update LokinetRouter.md

Syntax Correction

ChanoSan committed:
Update LokinetOverview.md

ChanoSan committed:
Update LokinetOverview.md

ChanoSan committed:
Update AccessingSNApps.md

ChanoSan committed:
Create Install.md

ChanoSan committed:
Update Install.md

ChanoSan committed:
Update Install.md

ChanoSan committed:
Update Install.md

ChanoSan committed:
Update Install.md

ChanoSan committed:
Update JoinOpenGroupMobile.md

ChanoSan committed:
Update JoinOpenGroupDesktop.md

ChanoSan committed:
Create PublicTestingGuide.md

ChanoSan committed:
Create GettingStarted.md

ChanoSan committed:
Merge pull request #12 from loki-project/master

Merge Upstream Changes

ChanoSan committed:
Update lokinet-windows-guide.md

ChanoSan committed:
Update PublicTestingGuide.md

ChanoSan committed:
Merge pull request #13 from loki-project/master

Add sudo to commands

ChanoSan committed:
Update Install.md

ChanoSan committed:
Merge pull request #14 from loki-project/master

Remove old rpc guides

ChanoSan committed:
Merge pull request #15 from loki-project/master

Merge Upstream Changes

ChanoSan committed:
Merge pull request #16 from oxen-io/master

Merge Upstream Changes
